src/App.css

```diff
@@ -1,3 +1,367 @@
-.App {
-  color: #000;
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Slab:wght@300;400;700&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.ff-m {
+  font-family: 'Montserrat', sans-serif;
+}
```

Comment:
- [ ] You can make this class a general font class which you will apply everywhere this font family is used so that you remove this rule from line 14, and line 270 where you are repeating it

Reply:
Done!